assignments/stretch-function-conversion.js

```diff
@@ -1,23 +1,35 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
 // let myFunction = function () {};
+let myFunction = () => {};
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
 
+let anotherFunction = () => param;
+
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
 
+let add = (param1, param2) => param1 + param2;
+add(1,2) 
+
 // let subtract = function (param1, param2) {
 //   return param1 - param2;
 // };
 // subtract(1,2);
+let subtract = (param1, param2) => param1 - param2;
+subtract(1,2);
 
 // exampleArray = [1,2,3,4];
 // const triple = exampleArray.map(function (num) {
 //   return num * 3;
 // });
-// console.log(triple);
\ No newline at end of file
+// console.log(triple);
+
```

Comment:
since you only have one parameter the parenthesis is optional, it's fine to leave it on just want to make sure you know about it.